### PR TITLE
#33 social impact beyond business

### LINKED ISSUE #33: 🎨 Create Social Impact beyond business section
#  🎨 Create Social Impact beyond business section
---

#### Note: This issue is dependent on #31, please complete that issue first

Quick links: [Summary](#summary) | [Description](#description) | [Technical-Analysis](#Technical-Analysis) | [Acceptance-Criteria](#Acceptance-Criteria)

---

### Summary

As a user I want a section that describes the organisation beyond business.

### Description

This component displays provided text and images side by side on the social impact section.

Then section has the following :

```
1. An image on the left side
2. A paragraph of text
3. A title

```

Screenshot 2023-02-06 at 10 24 53

Your assignment is to implement the component to appear as above, reuse the component on #31.

Please take note of all the design features (including mobile and rounded corners)

#### Link to full design:
```
https://xd.adobe.com/view/7b72572b-b40a-4618-bfa6-5500ab28f13b-2e21/screen/31438bb0-400a-4da8-a374-c0440535c37c

mobile 
-

#### Image link

https://res.cloudinary.com/dyl3rncv3/image/upload/v1675690301/elewa-group-website/Images/IMG_0764_twyo3k.png
```
### Technical Analysis

This component will be created in libs/pages/elewa/social-impact

### Acceptance Criteria

1. When rendered the component is pixel perfect compared to the design above.
2. The component is clean and does not have more than 50 lines
3. The component SCSS uses CSS variables for colour, font, ... Connected to the main style.scss
4. The code is isolated to the libs/page/home package, unless a justified reason is specified.
5. The component is aligned using CSS flex
6. The design is responsive for screen, tablet and phone
7. The image renders fast and has multiple srcsets

### PR DESCRIPTION
**Pull request by Bel-94**
This pull request is part of the work on issue #33,   which is dependent on issue #31 and #241 
To achieve this, I needed to:
```
- re-use the banner component #31
- import the BannerModule so that I can re-use the component
- Make it pixel perfect and mobile friendly
```
Fixes #33 
## Type of change
- [ ] New feature (non-breaking change which adds functionality)
# Screenshots

Social impact beyond business section with image on the left
Screenshot from 2023-02-22 15-53-15

Social impact beyond business section with image on the left mobile version
Screenshot from 2023-02-22 15-53-38

Screenshot from 2023-02-22 15-53-52

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] My changes generate no new warnings
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by Bel-94** [comment]
Hi @IanOdhiambo9 , @LemmyMwaura my new pull request #262 for issue #33

File: libs/pages/elewa/social-impact/src/lib/components/social-impact-beyond-business/social-impact-beyond-business.component.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'elewa-group-social-impact-beyond-business',
+  templateUrl: './social-impact-beyond-business.component.html',
+  styleUrls: ['./social-impact-beyond-business.component.scss'],
+})
+export class SocialImpactBeyondBusinessComponent {
+@Input() paragraphs = ["Some of that youthful energy is channeled beyond business. Through projects such as our coding school in Kakuma refugee camp, training teachers in pastoral Samburu and our open-source coding initiatives providing entry-level opportunities to hundreds of junior engineers.", "These projects don't necessarily have an immediate financial return, but their impact can be tangibly measured in the long-term."]
```
Comment (LemmyMwaura) [inline libs/pages/elewa/social-impact/src/lib/components/social-impact-beyond-business/social-impact-beyond-business.component.ts:9]: This should just be a property named paragraphs, no need for the Input decorator

**Review by LemmyMwaura — CHANGES_REQUESTED** [review]
Make the necessary updates and next time try and avoid making changes to files you are not using to satisfy the issue. Good job

File: libs/pages/elewa/social-impact/src/lib/components/social-impact-beyond-business/social-impact-beyond-business.component.ts (hunk 1 above)
Comment (Bel-94) [reply libs/pages/elewa/social-impact/src/lib/components/social-impact-beyond-business/social-impact-beyond-business.component.ts]: Hi @LemmyMwaura, I have changed the paragraph into a property and pushed the changes

Comment (LemmyMwaura) [reply libs/pages/elewa/social-impact/src/lib/components/social-impact-beyond-business/social-impact-beyond-business.component.ts]: Okay, Just sync the latest updates to resolve the conflicts

**Review by LemmyMwaura — APPROVED** [review]

**Comment by IanOdhiambo9** [comment]
This issue has been closed

**Pull request closed without merging**